hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/metrics/JRaftMetrics.java

```diff
@@ -172,35 +168,33 @@ private static void registerHistogram(String group, String name,
 
         String baseName = PREFIX + "." + name.toLowerCase();
 
-        HistogramWrapper wrapper = toWrapper(histogram);
-
-        Gauge.builder(baseName + ".median", wrapper, (d) -> d.getSnapshot().getMedian())
+        Gauge.builder(baseName + ".median", histogram, (d) -> d.getSnapshot().getMedian())
              .tags(tags).register(registry);
-        Gauge.builder(baseName + ".min", wrapper, (d) -> d.getSnapshot().getMin())
+        Gauge.builder(baseName + ".min", histogram, (d) -> d.getSnapshot().getMin())
              .tags(tags).register(registry);
-        Gauge.builder(baseName + ".max", wrapper, (d) -> d.getSnapshot().getMax())
+        Gauge.builder(baseName + ".max", histogram, (d) -> d.getSnapshot().getMax())
```

Comment:
A further question, if each dimension like `min/max/mean` needs to be retrieved getSnapshot() once, does it need to be optimized?

Reply:
No, each time you pull indicators, you need to put the latest one into the indicator system.